<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DT_SetupDlg</class>
 <widget class="QDialog" name="DT_SetupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>钉钉配置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DT_SetupDlg{background-image: url(:/image/box_back.jpg);}</string>
  </property>
  <widget class="QPushButton" name="testBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>推送信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="webhookEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="title">
    <string>通知用户</string>
   </property>
   <widget class="QRadioButton" name="noRb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>不通知</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="allRb">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>通知所有人</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="oneRb">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>通知指定用户</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobEdit_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>18576689472</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="enBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>关闭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>开启</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="keyEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>报警提醒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Webhook</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>关键词</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
